<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sistem Tracking Corona</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 42, 54)</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_top">
     <property name="minimumSize">
      <size>
       <width>1920</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(68, 71, 90)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_side">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>1080</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(68, 71, 90)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="frame_menu_2">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>1080</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(68, 71, 90)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_suhu">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Nirmala UI</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(68, 71, 90);
	border: 0px solid;
	text-align: left;
	padding: 10px
}
QPushButton:hover {
	background-color:rgb(255, 121, 198);
}</string>
           </property>
           <property name="text">
            <string>Suhu Tubuh</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_harian">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Nirmala UI</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(68, 71, 90);
	border: 0px solid;
	text-align: left;
	padding: 10px
}
QPushButton:hover {
	background-color:rgb(255, 121, 198);
}</string>
           </property>
           <property name="text">
            <string>Kasus Harian COVID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_pesan">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Nirmala UI</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(68, 71, 90);
	border: 0px solid;
	text-align: left;
	padding: 10px
}
QPushButton:hover {
	background-color:rgb(255, 121, 198);
}</string>
           </property>
           <property name="text">
            <string>Pesan Rumah Sakit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_member">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Nirmala UI</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(68, 71, 90);
	border: 0px solid;
	text-align: left;
	padding: 10px
}
QPushButton:hover {
	background-color:rgb(255, 121, 198);
}</string>
           </property>
           <property name="text">
            <string>Membership</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_logout">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Nirmala UI</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 85, 85);
	border: 0px solid;
	text-align: left;
	padding: 10px
}
QPushButton:hover {
	background-color:rgb(255, 100, 115);
}</string>
           </property>
           <property name="text">
            <string>Log Out</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QFrame" name="frame_5">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>138</width>
              <height>397</height>
             </rect>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_6">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>1920</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../img/nopicture.png</pixmap>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QLabel" name="lbl_info">
        <property name="font">
         <font>
          <family>Nirmala UI Semilight</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(139, 233, 253); text-align: center;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
